crates/test_utils/src/newman_runner.rs

```diff
@@ -0,0 +1,161 @@
+use std::{env, process::Command};
+
+use clap::{arg, command, Parser};
+use masking::PeekInterface;
+
+use crate::connector_auth::{ConnectorAuthType, ConnectorAuthenticationMap};
+
+#[derive(Parser)]
+#[command(version, about = "Postman collection runner using newman!", long_about = None)]
+struct Args {
+    /// Admin API Key of the environment
+    #[arg(short, long = "admin_api_key")]
+    admin_api_key: String,
+    /// Base URL of the Hyperswitch environment
+    #[arg(short, long = "base_url")]
+    base_url: String,
+    /// Name of the connector
+    #[arg(short, long = "connector_name")]
+    connector_name: String,
+    /// Folder name of specific tests
+    #[arg(short, long = "folder")]
+    folder_s: Option<String>,
```

Comment:
Why not name this as `folder` itself?

Reply:
I intentionally added that `_s` to address multiple folders